### PR TITLE
Onboarding Cluster-logging, Elasticsearch and Loki operator on ROSA Classic

### PR DESCRIPTION
**Pull request by Sau1506mya**
Link to JIRA : [[LPTOCPCI-837](https://issues.redhat.com/browse/LPTOCPCI-837)]

### CODE REVIEW

**Comment by Sau1506mya** [comment]
/pj-rehearse periodic-ci-openshift-logging-interop-ocp-ci-main-cluster-logging-ocp4.15-lp-rosa-classic-cluster-logging-operator-rosa-aws

**Comment by Sau1506mya** [comment]
/pj-rehearse periodic-ci-openshift-logging-interop-ocp-ci-main-cluster-logging-ocp4.15-lp-rosa-classic-cluster-logging-operator-rosa-aws

File: ci-operator/config/openshift-logging/interop-ocp-ci/openshift-logging-interop-ocp-ci-main__cluster-logging-ocp4.15-lp-rosa-classic.yaml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      memory: 4Gi
+tests:
+- as: cluster-logging-operator-rosa-aws
+  cron: 0 6 25 10 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      CLO_SUB_CHANNEL: stable-5.9
+      CLO_SUB_SOURCE: redhat-operators
+      E2E_RUN_TAGS: '@level0'
```
Comment (anpingli) [inline ci-operator/config/openshift-logging/interop-ocp-ci/openshift-logging-interop-ocp-ci-main__cluster-logging-ocp4.15-lp-rosa-classic.yaml:60]: Can you update the E2E_RUN_TAGS to @rosa|@level0?

**Comment by Sau1506mya** [comment]
/pj-rehearse periodic-ci-openshift-logging-interop-ocp-ci-main-cluster-logging-ocp4.15-lp-rosa-classic-cluster-logging-operator-rosa-aws

**Comment by Sau1506mya** [comment]
/pj-rehearse periodic-ci-openshift-logging-interop-ocp-ci-main-cluster-logging-ocp4.16-lp-rosa-classic-cluster-logging-operator-rosa-aws

**Pull request merged**